### PR TITLE
119 continue translations

### PR DESCRIPTION
**Pull request by mabelene**
@LemonFace0309 would you like me to also add the typography changes for spaces here? I added it as a new issue bc i didn't want the review to be 100000 lines of code

### CODE REVIEW

**Comment by LemonFace0309** [comment]
> @LemonFace0309 would you like me to also add the typography changes for spaces here? I added it as a new issue bc i didn't want the review to be 100000 lines of code

wait wdym by typography changes? Lol

File: components/Dashboard/Modals/ProfileDialog/AccountDetails.js [hunk 1]
```diff
@@ -96,10 +99,10 @@ const AccountDetails = ({ session, editMode, saveChanges, setSaveChanges, setEdi
         if (newImage) await handleUpdateImage();
         setSaveChanges(false);
         if (!passed) {
-          alert(message ?? 'You account info has been updated sucessfully 😃');
+          alert(message ?? t('SUCCESS_CHANGE_PROFILE'));
           setEditMode(true);
         } else {
-          alert(message ?? 'Internal Server Error');
+          alert(message ?? t('ERROR_SERVER'));
```
Comment (LemonFace0309) [inline components/Dashboard/Modals/ProfileDialog/AccountDetails.js:105]: can you prefix with `LABEL_` ?

File: components/Dashboard/Modals/ProfileDialog/ChangePassword.js [hunk 2]
```diff
@@ -76,13 +78,13 @@ const ChangePassword = ({ session, editMode, saveChanges, setSaveChanges }) => {
     <Grid container direction="column" spacing={3}>
       <Grid item xs={12} md={6}>
         <Typography variant="h6" className={classes.title}>
-          Enter your current and new password
+          {t('PROMPT_PASSWORD')}
```
Comment (LemonFace0309) [inline components/Dashboard/Modals/ProfileDialog/ChangePassword.js:81]: same here

File: components/Dashboard/Modals/ProfileDialog/ChangePassword.js [hunk 3]
```diff
@@ -54,20 +56,20 @@ const ChangePassword = ({ session, editMode, saveChanges, setSaveChanges }) => {
...
-      alert(err?.response?.data?.message ?? 'Internal Server Error');
+      alert(err?.response?.data?.message ?? t('ERROR_SERVER'));
     }
   };
 
   if (!session?.user.type || session?.user.type !== 'credentials') {
     return (
       <Typography variant="h6" className={classes.title}>
-        Cannot change password for users signed up through third party services like Google or Facebook.
+        {t('ERROR_THIRD_PARTY_PASSWORD')}
```
Comment (LemonFace0309) [inline components/Dashboard/Modals/ProfileDialog/ChangePassword.js:72]: same here

File: components/Dashboard/Modals/ProfileDialog/ChangePassword.js [hunk 4]
```diff
@@ -54,20 +56,20 @@ const ChangePassword = ({ session, editMode, saveChanges, setSaveChanges }) => {
...
       console.debug(response);
-      alert(response?.data.message ?? 'Password updated succesfully 😃');
+      alert(response?.data.message ?? t('SUCCESS_UPDATE_PASSWORD'));
       setCurrentPassword('');
       setNewPassword1('');
       setNewPassword2('');
     } catch (err) {
       setServerError(true);
-      alert(err?.response?.data?.message ?? 'Internal Server Error');
+      alert(err?.response?.data?.message ?? t('ERROR_SERVER'));
```
Comment (LemonFace0309) [inline components/Dashboard/Modals/ProfileDialog/ChangePassword.js:65]: same here

File: components/Dashboard/Modals/ProfileDialog/ChangePassword.js [hunk 5]
```diff
@@ -54,20 +56,20 @@ const ChangePassword = ({ session, editMode, saveChanges, setSaveChanges }) => {
         },
       });
       console.debug(response);
-      alert(response?.data.message ?? 'Password updated succesfully 😃');
+      alert(response?.data.message ?? t('SUCCESS_UPDATE_PASSWORD'));
```
Comment (LemonFace0309) [inline components/Dashboard/Modals/ProfileDialog/ChangePassword.js:59]: same here

**Comment by mabelene** [comment]
> @LemonFace0309 would you like me to also add the typography changes for spaces here? I added it as a new issue bc i didn't want the review to be 100000 lines of code



> > @LemonFace0309 would you like me to also add the typography changes for spaces here? I added it as a new issue bc i didn't want the review to be 100000 lines of code
> 
> @mabelene wait wdym by typography changes? Lol

@LemonFace0309 oops mb, i meant translations

File: components/Dashboard/Sidebar.js [hunk 6]
```diff
@@ -1,9 +1,10 @@
 /* eslint-disable jsx-a11y/accessible-emoji */
+import { useTranslation } from 'next-i18next';
+
```
Comment (LemonFace0309) [inline components/Dashboard/Sidebar.js:3]: delete line 3 pls

**Review by LemonFace0309 — APPROVED** [review]
LGTM, just make changes I suggested, click on resolve conversation when you're done, and ensure it works properly for different locales. :D

**Comment by LemonFace0309** [comment]
> > @LemonFace0309 would you like me to also add the typography changes for spaces here? I added it as a new issue bc i didn't want the review to be 100000 lines of code
> 
> > > @LemonFace0309 would you like me to also add the typography changes for spaces here? I added it as a new issue bc i didn't want the review to be 100000 lines of code
> > 
> > 
> > @mabelene wait wdym by typography changes? Lol
> 
> @LemonFace0309 oops mb, i meant translations

if you want. I don't mind. translations are ezpz to review. Just rerequest a review from me if you do. 
image
(the circle icon besides the green checkmark)

File: components/Dashboard/Modals/ProfileDialog/AccountDetails.js (hunk 1 above)
Comment (mabelene) [reply components/Dashboard/Modals/ProfileDialog/AccountDetails.js:105]: okk do u want all translations to be prefixed with LABEL_ ?

Comment (LemonFace0309) [reply components/Dashboard/Modals/ProfileDialog/AccountDetails.js]: Yes pls :3

**Pull request merged**